ckb-transaction-cobuild/src/lib.rs

```diff
@@ -99,10 +99,16 @@ pub fn check_others_in_group() -> Result<(), Error> {
     }
 }
 
-pub fn generate_signing_message_hash(message: &Message) -> Result<[u8; 32], Error> {
-    let mut hasher = new_sighash_all_blake2b();
+fn generate_signing_message_hash(message: &Option<Message>) -> Result<[u8; 32], Error> {
     // message
-    hasher.update(message.as_slice());
+    let mut hasher = match message {
```

Comment:
For lock script, it is not necessary to get message. When a message is too large, it consumes a lot of memory instantly. It would be better to use batching update hash. This function accepts no argument.